hooks/MWD.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+/**
+ * Copyright 2016 Signal K and Fabian Tollenaar <fabian@signalk.org>.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const debug = require('debug')('signalk-parser-nmea0183/MWD')
+const utils = require('@signalk/nmea0183-utilities')
+
+/*
+ * $WIMWD,<0>,<1>,<2>,<3>,<4>,<5>,<6>,<7>*hh
+ *
+ * NMEA 0183 standard Wind Direction and Speed, with respect to north.
+ *
+ * <0> Wind direction, 0.0 to 359.9 degrees True, to the nearest 0.1 degree
+ * <1> T = True
+ * <2> Wind direction, 0.0 to 359.9 degrees Magnetic, to the nearest 0.1 degree
+ * <3> M = Magnetic
+ * <4> Wind speed, knots, to the nearest 0.1 knot.
+ * <5> N = Knots
+ * <6> Wind speed, meters/second, to the nearest 0.1 m/s.
+ * <7> M = Meters/second
+ */
+
+module.exports = function(input) {
+  const { id, sentence, parts, tags } = input
+  var pathValues = []
+
+  if(parts[0] != ''){
```

Comment:
Since the sentence structure seems to provide for having the fields in different order I'd like to include also tests for the fields 1 and 3.

Reply:
Ok, I added the tests for all the fields. 
Are you sure the fields are allowed in different order?  From the definition I would assume the fields (data/type) for True/Magnetic/Knots/Meters per Second must be in exactly this sequence.